src/top_model/main.cpp

```diff
@@ -56,6 +60,10 @@ class ApplicationGen : public iestream_input<message_t,T> {
 
 int main(int argc, char ** argv){
 
+	const char *input_file = TOP_MODEL_OUTPUT;
```

Comment:
as per naming convention , pointer variable should start with "p_", 
Please check for the same in all the files